### ISSUE TITLE
Crash in rasterizer while using the GridMap plugin. Callstack below.

### ISSUE THREAD

**Issue by PeaceSells50** [godotengine/godot#2139]
Windows 8.1. Surface 2 Pro. 64bit Debug.

```
Callstack:
 	ig7icd32.dll!53544384()	Unknown
 	[Frames below may be incorrect and/or missing, no symbols loaded for ig7icd32.dll]	
 	ig7icd32.dll!5351fe0b()	Unknown
>	godot.windows.tools.64.exe!RasterizerGLES2::_canvas_item_render_commands<0>(Rasterizer::CanvasItem * p_item, Rasterizer::CanvasItem * current_clip, bool & reclip) Line 9040	C++
 	godot.windows.tools.64.exe!RasterizerGLES2::canvas_render_items(Rasterizer::CanvasItem * p_item_list, int p_z, const Color & p_modulate, Rasterizer::CanvasLight * p_light) Line 9449	C++
 	godot.windows.tools.64.exe!VisualServerRaster::_render_canvas(VisualServerRaster::Canvas * p_canvas, const Matrix32 & p_transform, Rasterizer::CanvasLight * p_lights) Line 6907	C++
 	godot.windows.tools.64.exe!VisualServerRaster::_draw_viewport(VisualServerRaster::Viewport * p_viewport, int p_ofs_x, int p_ofs_y, int p_parent_w, int p_parent_h) Line 7159	C++
 	godot.windows.tools.64.exe!VisualServerRaster::_draw_viewports() Line 7291	C++
 	godot.windows.tools.64.exe!VisualServerRaster::draw() Line 7377	C++
 	godot.windows.tools.64.exe!VisualServerWrapMT::draw() Line 136	C++
 	godot.windows.tools.64.exe!Main::iteration() Line 1404	C++
 	godot.windows.tools.64.exe!OS_Windows::run() Line 2239	C++
 	godot.windows.tools.64.exe!widechar_main(int argc, wchar_t * * argv) Line 146	C++
 	godot.windows.tools.64.exe!main(int _argc, char * * _argv) Line 172	C++
 	[External Code]	
```

**Comment by akien-mga** [comment]
Can you still reproduce this issue in the current master branch? It might have been related to your other GridMap crash #2140, and therefore potentially fixed by #2165 too.

**Comment by akien-mga** [comment]
Assuming this was fixed as per the above comment.
